data/yam/agama/auto/sle_lvm.jsonnet

```diff
@@ -0,0 +1,36 @@
+{
+  "product": {
+    "id": "SLES_16.0"
+  },
+  "user": {
+    "fullName": "Bernhard M. Wiedemann",
+    "password": "$6$vYbbuJ9WMriFxGHY$gQ7shLw9ZBsRcPgo6/8KmfDvQ/lCqxW8/WnMoLCoWGdHO6Touush1nhegYfdBbXRpsQuy/FTZZeg7gQL50IbA/",
+    "hashedPassword": true,
+    "userName": "bernhard"
+  },
+  "root": {
+    "password": "$6$vYbbuJ9WMriFxGHY$gQ7shLw9ZBsRcPgo6/8KmfDvQ/lCqxW8/WnMoLCoWGdHO6Touush1nhegYfdBbXRpsQuy/FTZZeg7gQL50IbA/",
+    "hashedPassword": true
+  },
+  "storage": {
+    "guided": {
+      "target": {
+        "newLvmVg": [
+          "/dev/vda"
```

Comment:
pvm uses sda, so I'm not sure how this could work.

Reply:
Thanks, then how about s390x?  Either vad or sda works for s390x.

Reply:
https://suse.slack.com/archives/C02TLF25571/p1736250075337299

Reply:
we can use this:
https://suse.slack.com/archives/C02TLF25571/p1736251663564089?thread_ts=1736250075.337299&cid=C02TLF25571